dm.js.org

bullwinkle opened the pull request:
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. But it looks like your page is not configured correctly. I would guess that you are using the "master" branch for GitHub pages but the CNAME is in the "gh-pages"branch. You have to fix this to make it work. 
FYI: https://help.github.com/articles/configuring-a-publishing-source-for-github-pages
I'll merge after we have "contact".

bullwinkle commented:
> I've already added your requested subdomain to JS.ORGs zonefile. But it looks like your page is not configured correctly. I would guess that you are using the "master" branch for GitHub pages but the CNAME is in the "gh-pages"branch. You have to fix this to make it work.
> FYI: https://help.github.com/articles/configuring-a-publishing-source-for-github-pages
> I'll merge after we have "contact".

Thanks a lot!
You are right, I've changed the source of github pages to root directory of 'gh-pages' branch.

indus commented:
Welcome https://dm.js.org

indus commented:
@bullwinkle While the CNAME now works. I wonder if this fullfills the requirement of "reasonable content". I've marked thsi as provisional to make sure to check back and see if there is more the 4 partially working links.
